### PR TITLE
Adds adrenaline-pumping super epic ultra time music to the declaration of war

### PR DESCRIPTION
**Pull request by swissloaf**
# General Documentation

### Intent of your Pull Request
Adds music to when there is a declaration of war

https://www.youtube.com/watch?v=H8ZCM4FuI5A

This is the song. The video is 2 minutes but the actual file fades out at around 55 seconds

Credit to Joel Nielsen for producing it.

### Why is this change good for the game?
Good ambience, gets your adrenaline pumping

# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 
This adds music to the declaration of war, when it is declared.

### What should players be aware of when it comes to the changes your PR is implementing?
It is music that plays.

### What general grouping does this PR fall under? 
Sound

### If there are any numerical values involved in your PR that will be relevant to a player, please note them here. 

# Changelog

:cl:  
soundadd: Adds music to the declaration of war
/:cl:

### CODE REVIEW

**Comment by ToasterBiome** [comment]
Damn this is kinda pog as fuck...

**Comment by UselessTheremin** [comment]
We already got enough half life imo if that played as one of multiple songs I’d be more okay with it, I have some that I can add at a later date

**Comment by swissloaf** [comment]
> We already got enough half life imo if that played as one of multiple songs I’d be more okay with it, I have some that I can add at a later date

is black mesa

**Comment by ChesterTheCheesy** [comment]
eh? don't you think a 2 minute video is an overkill?

**Comment by ChesterTheCheesy** [comment]
Also, did you test if it doesn't immediately lag the game making an instant 30% dilation?

**Comment by UselessTheremin** [comment]
> > We already got enough half life imo if that played as one of multiple songs I’d be more okay with it, I have some that I can add at a later date
> 
> is black mesa

Dude close enough it’s really not my thing

**Comment by swissloaf** [comment]
Epic

**Comment by swissloaf** [comment]
> eh? don't you think a 2 minute video is an overkill?

good point. I reduce to 30 second with fade out

> Also, did you test if it doesn't immediately lag the game making an instant 30% dilation?

i test all my prs mane......

**Comment by swissloaf** [comment]
> > > We already got enough half life imo if that played as one of multiple songs I’d be more okay with it, I have some that I can add at a later date
> > 
> > 
> > is black mesa
> 
> Dude close enough it’s really not my thing

well golleee we might as well call this useless the remin station

**Comment by UselessTheremin** [comment]
> > > > We already got enough half life imo if that played as one of multiple songs I’d be more okay with it, I have some that I can add at a later date
> > > 
> > > 
> > > is black mesa
> > 
> > 
> > Dude close enough it’s really not my thing
> 
> well golleee we might as well call this useless the remin station

Whatever man. I am giving my opinion and didn’t claim otherwise, am I supposed to be a collective and speak for yogstation? No. My point is that for me personally I don’t care about the subject matter and I feel it’s a bit odd for it to be for the declaration of war, idk the music isn’t my favorite thing. That’s all I wanted to say.

**Comment by swissloaf** [comment]
> > > > > We already got enough half life imo if that played as one of multiple songs I’d be more okay with it, I have some that I can add at a later date
> > > > 
> > > > 
> > > > is black mesa
> > > 
> > > 
> > > Dude close enough it’s really not my thing
> > 
> > 
> > well golleee we might as well call this useless the remin station
> 
> Whatever man. I am giving my opinion and didn’t claim otherwise, am I supposed to be a collective and speak for yogstation? No. My point is that for me personally I don’t care about the subject matter and I feel it’s a bit odd for it to be for the declaration of war, idk the music isn’t my favorite thing. That’s all I wanted to say.

sound is a critical part of game design and nothing is better than a pre-sequence sound file that gets your adrenaline pumping for the big fight that is the nuke ops

**Comment by swissloaf** [comment]
github lurkers would you rather hear the one listed above or https://www.youtube.com/watch?v=30AM8K06FJU or https://www.youtube.com/watch?v=DPqXT4dyfNY

**Comment by ToasterBiome** [comment]
> github lurkers would you rather hear the one listed above or https://www.youtube.com/watch?v=30AM8K06FJU

Resident Lurker here. I like the first one better, I think the faster tempo guitar really hit home with it. I mean this is very nice, but it has less energy.

**Comment by swissloaf** [comment]
> > github lurkers would you rather hear the one listed above or https://www.youtube.com/watch?v=30AM8K06FJU
> 
> Resident Lurker here. I like the first one better, I think the faster tempo guitar really hit home with it. I mean this is very nice, but it has less energy.

I added another one. Check it out (same comment you replied too). I really like that one too

**Comment by ToasterBiome** [comment]
Ye, We've Got Hostiles vs Surface Tension 2? I like We've Got Hostiles more.

**Comment by swissloaf** [comment]
> Ye, We've Got Hostiles vs Surface Tension 2? I like We've Got Hostiles more.

https://www.youtube.com/watch?v=DPqXT4dyfNY is the one I was referring to

**Comment by ToasterBiome** [comment]
> > Ye, We've Got Hostiles vs Surface Tension 2? I like We've Got Hostiles more.
> 
> https://www.youtube.com/watch?v=DPqXT4dyfNY is the one I was referring to

Yuck Github didnt update the comment until I CTRL+F5, not just F5. 

Gah,, Klaxon Beat is so iconic. It's a tough choice. Klaxon Beat for the meme but We've got Hostiles for the epicness. As for SS13 I don't know which one would be better!

**Comment by Xoxeyos** [comment]
I don't think we need music for a war operation.

**Comment by ToasterBiome** [comment]
> I don't think we need music for a war operation.

Hypes up the moment, one of the few times that the crew can all come together (or not!) to do something..

**Comment by JamieD1** [comment]
Bit long, Make it 20-30 seconds max.

**Comment by swissloaf** [comment]
> Bit long, Make it 20-30 seconds max.

Will take a look later today

**Comment by swissloaf** [comment]
Added additonal music

**Comment by JamieD1** [comment]
@swissloaf Still not 20-30 seconds max.
image

**Review by JamieD1 — CHANGES_REQUESTED** [review]
20-30 Seconds max.

**Comment by swissloaf** [comment]
> 20-30 Seconds max.

Will do in a couple of days however that 1 minute one should stay as 1 minute unless this is a file size limitation. It's not overtly intrusive like the 42 second one. I will try to reduce the 42 second one however but a lot of the bass kicks off at around that time which is why I chose 42 seconds
